src/styles/components/_code-pattern.scss

```diff
@@ -0,0 +1,37 @@
+@import '../settings/colors';
+
+// =============================================================================
+// CODE PATTERN component
+//
+// A component that displays the code and a demo side by side.
+//
+// =============================================================================
+
+.code-pattern {
+  width: 100%;
+
+  .code-pattern__content {
+    display: grid;
+    gap: 1rem; // sass-lint:disable-line no-misspelled-properties
+    grid-template-columns: 50% auto;
+    justify-items: stretch;
+    height: 300px;
```

Comment:
Probably don't want this here. I think we may instead give the iframe an explicit size (and maybe make it configurable by the author somehow?) and let that define the overall height for the container.

Reply:
It is causing a CLS if there's no height defined.

Reply:
right I meant you would give the iframe the explicit size, not the parent

Reply:
I guess this works too if you want to leave it like this. But we'll need to come up with a way for this to be configurable by the author. One option is for the shortcode to accept a size which it then applies to the element as an inline style